lib/lightning/workflows/edge.ex

```diff
@@ -47,7 +47,7 @@ defmodule Lightning.Workflows.Edge do
 
     field :delete, :boolean, virtual: true
 
-    timestamps()
+    timestamps(type: :utc_datetime)
```

Comment:
It has not failed but on the database it's still without time zone:
![Screenshot from 2024-04-04 12-29-18](https://github.com/OpenFn/lightning/assets/44991200/482c01a7-6efb-45cd-880a-636b8873523d)

Reply:
That's the expected type, Ecto's `utc_datetime` types in migrations don't use a timezone option in column types. The application/repo handles the tz conversion.

It's a tricky path to deviate from, having the db set to another tz, or have Ecto store as a different one. So we're using the defaults provided by Ecto, everything in UTC via Ecto and if we want to do logic/presentation in a different tz it all happens in queries or frontend functions.